src/main/java/nl/b3p/tailormap/api/controller/UserController.java

```diff
@@ -45,4 +51,23 @@ public ResponseEntity<Serializable> getUser() {
     }
     return ResponseEntity.status(HttpStatus.OK).body(userResponse);
   }
+
+  @GetMapping(
+      path = "${tailormap-api.base-path}/login/configuration",
+      produces = MediaType.APPLICATION_JSON_VALUE)
+  @SuppressWarnings("unchecked")
+  public ResponseEntity<LoginConfiguration> getSSOEndpoints() {
+    LoginConfiguration result = new LoginConfiguration();
+
+    if (clientRegistrationRepository instanceof Iterable<?>) {
```

Comment:
We can inject our own `OIDCRespository` with its specific type which is explicitly `Iteratable` so this cast isn't necessary (see [here](https://github.com/spring-projects/spring-security/issues/5917))

Reply:
This code was written this sway to provide (superficial?) support for OAuth2/OIDC connections configured via the `application.properties` - though i suspect that's probably not going to be used at all at this point..